composer.json

```
@@ -1,66 +1,156 @@
 {
 	"name": "pfefferle/wordpress-activitypub",
-	"description": "The ActivityPub protocol is a decentralized social networking protocol based upon the ActivityStreams 2.0 data format.",
+	"description": "A WordPress implementation of the ActivityPub protocol, enabling decentralized social networking using the ActivityStreams 2.0 data format.",
 	"type": "wordpress-plugin",
+	"keywords": [
+		"wordpress",
+		"activitypub",
+		"fediverse",
+		"decentralized",
+		"social-network"
+	],
+	"homepage": "https://github.com/pfefferle/wordpress-activitypub",
```

The main repo is this one now.

```suggestion
	"homepage": "https://github.com/Automattic/wordpress-activitypub",
```